### PR TITLE
Add docs for managed integrations endpoint

### PR DESCRIPTION
**Pull request by Neeraj-Ghodla**

### CODE REVIEW

File: src/api/integrations/managed-integrations.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+| :----- | :------ | :----------------------------------------------------------- |
+| active | Boolean | Flag indicating whether the Managed Integration is active.   |
+| params | Object  | [Params](#params) depending on the Managed Integration type. |
+
+{% h4 Example response payload %}
+
+{% json %}
+{
+  "id": "5ee0c486308f590260d9a07f",
+  "type": "kete-enrolment",
```
Comment (MeganSteenkamp) [inline src/api/integrations/managed-integrations.md:91]: TODO: Remove type from response

File: src/api/integrations/managed-integrations.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+| :----------------- | :----------------- | :------------------------------------------------------------------------------------------------------------- |
+| id                 | String             | The Managed Integration's unique identifier.                                                                   |
+| type               | String             | The type of Managed Integration. Supported types are `kete-enrolment`.                                         |
+| active             | Boolean            | Flag indicating whether the Managed Integration is active.                                                     |
+| state              | String             | Current state of the Managed Integration. Supported values are `pending`, `active`, `inactive` and `archived`. |
+| externalId         | String             | Field used to reference an id from an external system.                                                         |
+| claimedByAccountId | String             | Centrapay account id used to claim the Managed Integration.                                                    |
```
Comment (IsaacMurtagh) [inline src/api/integrations/managed-integrations.md:31]: Can we remove claimedByAccountId, claimedBy and claimedAt from mandatory fields, if it hasn't been claimed yet, these won't exist

File: src/api/integrations/managed-integrations.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+## Models
+
+### Managed Integration
+
+{% h4 Mandatory Fields %}
+
+|       Field        |        Type        |                                                  Description                                                   |
+| :----------------- | :----------------- | :------------------------------------------------------------------------------------------------------------- |
+| id                 | String             | The Managed Integration's unique identifier.                                                                   |
+| type               | String             | The type of Managed Integration. Supported types are `kete-enrolment`.                                         |
```
Comment (IsaacMurtagh) [inline src/api/integrations/managed-integrations.md:27]: If we're exposing the possible types can we add a table for them

**Review by IsaacMurtagh — CHANGES_REQUESTED** [review]

File: src/api/integrations/managed-integrations.md (hunk 3 above)
Comment (MeganSteenkamp) [reply src/api/integrations/managed-integrations.md:27]: I have addressed this.

Comment (MeganSteenkamp) [reply src/api/integrations/managed-integrations.md:27]: Is there a reason for this? I don't think this is currently necessary.

Comment (IsaacMurtagh) [reply src/api/integrations/managed-integrations.md:27]: I'm thinking It won't be long until more types are here seeing it's needed by a few projects. I'd say it's good to get most of the work done whilst you're working on it to reduce rework. But if you think it should be done later then that's fine too.

**Review by IsaacMurtagh — COMMENTED** [review]
LGTM now :) - I'll let someone else approve though

**Review by IsaacMurtagh — APPROVED** [review]

**Review by GeekOnGadgets — APPROVED** [review]

**Pull request merged**